### PR TITLE
javascript-modal

### PR DESCRIPTION
**Pull request by bernardino8**
https://user-images.githubusercontent.com/88168364/131591911-62446380-1d17-4195-a1e6-d57850960eef.mp4

### CODE REVIEW

File: javascript-modal/main.js [hunk 1]
```diff
@@ -1,17 +1,17 @@
-var modal = document.querySelector('.modal');
+var modalWindow = document.querySelector('.modal');
 var openButton = document.querySelector('.open-modal-button');
 var closeButton = document.querySelector('.close-modal-button');
 
 openButton.onclick = function () {
```
Comment (codingcodymiller) [inline javascript-modal/main.js:5]: You definitely shouldn't apply event handlers this way. Please use the `addEventListener` method I taught you, this method of applying event handlers is somewhat outdated.

File: javascript-modal/main.js [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-  modal.style.display = 'block';
+  modalWindow.style.display = 'block';
 };
 
 closeButton.onclick = function () {
-  modal.style.display = 'none';
+  modalWindow.style.display = 'none';
```
Comment (codingcodymiller) [inline javascript-modal/main.js:10]: When you assign values directly to properties of the `style` property on elements, you are creating inline styling, which is a big no-no. Instead, you should prepare classes in your stylesheet that you can add and remove as necessary.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start, man! A couple notes for you:

**Comment by bernardino8** [comment]
https://user-images.githubusercontent.com/88168364/131959291-a565a2c4-65a5-4b0a-88f3-e17c5bc6549d.mp4
thought of using event delegation for this assignment but opted not to; wonder how it would like though

**Review by codingcodymiller — APPROVED** [review]
Nice work!! Since there isn't anything being dynamically generated in this, it wouldn't be a great opportunity for event delegation, but you can give it a shot for practice!

**Pull request merged**